format/spec.md

```diff
@@ -444,7 +459,7 @@ Partition field IDs must be reused if an existing partition spec contains an equ
 
 | Transform name    | Description                                                  | Source types                                                                                              | Result type |
 |-------------------|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|-------------|
-| **`identity`**    | Source value, unmodified                                     | Any                                                                                                       | Source type |
+| **`identity`**    | Source value, unmodified                                     | Any other than `variant`                                                                                  | Source type |
```

Comment:
What out "except for"? I think that is more clear than "other than"

Reply:
Should we also fix the `bucket` function definition by using an `except for` list?

Reply:
Make sense. Let me fix in the following PR.
> Should we also fix the `bucket` function definition by using an `except for` list?